tasks/lets_encrypt.yml

```diff
@@ -63,7 +63,7 @@
     challenge: "{{ lets_encrypt_challenge_type }}"
     agreement: "{{ lets_encrypt_agreement }}"
     csr: "/etc/ssl/requests/{{ lets_encrypt_resource_name }}.csr"
-    dest: "/etc/ssl/certs/{{ lets_encrypt_resource_name }}.crt"
+    dest: "{{ build_dir.path }}/01-{{ lets_encrypt_resource_name }}.crt"
```

Comment:
please provide a default value for the variable under `defaults/main.yml`

Reply:
Not required as this variable is filled during the runtime from the script and not ment to be changed from the outside of the script.

For refernce have look at line 11:
https://github.com/sitewards/ansible-role-lets-encrypt/pull/11/files/5571c3f2e5148d72fea54a2693a0a7d1472aad1e..ef7871590df0e2464af73d04b433f122ef6865cb#diff-7a113781da4adef4962744aaf7fb9a56L11